WoWsShipBuilder.UI/ViewModels/ShipSelectionWindowViewModel.cs

```diff
@@ -34,6 +36,13 @@ public ShipSelectionWindowViewModel(Window? win)
             Dictionary<string, ShipSummary> shipNameDictionary = AppData.ShipSummaryList.ToDictionary(ship => Localizer.Instance[$"{ship.Index}_FULL"].Localization, ship => ship);
             FilteredShipNameDictionary = new SortedDictionary<string, ShipSummary>(shipNameDictionary);
             SummaryList = new AvaloniaList<KeyValuePair<string, ShipSummary>>(FilteredShipNameDictionary.Select(entry => entry));
+            multiSelectionEnabled = multiSelection;
+            SelectedShipList.CollectionChanged += SelectedShipList_CollectionChanged;
+        }
+
+        private void SelectedShipList_CollectionChanged(object? sender, System.Collections.Specialized.NotifyCollectionChangedEventArgs e)
+        {
+            this.RaisePropertyChanged(nameof(SelectedShipList));
```

Comment:
Bad design. Try to properly use the CollectionChanged events. However, due to command bindings, this may be the only way.